src/main/java/org/spongepowered/api/entity/hanging/Hanging.java

```diff
@@ -43,7 +43,7 @@
      * Sets the facing direction of this entity.
      *
      * @param direction The direction to face
-     * @param forced Whether physics is ignored
+     * @param forced    Whether physics is ignored
```

Comment:
Maybe. But those extra formatting just blows up the entire PR. Although some of them may be useful.
(You/the team made the rules, it would be very nice to know those rules apply to Teammembers as well.)

IMHO @simon816 is right with this one.